indexer/osm_editor.cpp

```diff
@@ -221,7 +221,20 @@ void Editor::LoadMapEdits()
           }
           else
           {
-            fti.m_feature = *m_getOriginalFeatureFn(fid);
+            auto const originalFeaturePtr = m_getOriginalFeatureFn(fid);
+            if (!originalFeaturePtr)
+            {
+              LOG(LERROR, ("A feature appears missing/deleted while loading edits."));
```

Comment:
Feel free to leave this as is, but I'm against such long comments. This comment can be much simpler (with the same amount of information):
```c++
LOG(LERROR, ("A feature can't be loaded:", fid));
```